test/src/e2e_vm_tests/test_programs/should_pass/language/tuple_single_element/Forc.toml

```diff
@@ -0,0 +1,6 @@
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+license = "Apache-2.0"
+name = "tuple_single_element"
+entry = "main.sw"
+implicit-std = true
```

Comment:
The test should use a path to `std`. Or, as Alex said, maybe avoid using `assert` altogether and set this to `false`.

Reply:
> The test should use a path to std

Why is that?

Reply:
Ah I see, makes sense! But instead of pointing it to GitHub at all, its probably better to require that all tests use the local version of `core` and `std`.